CONTRIBUTING.md

```diff
@@ -45,7 +45,7 @@ tox run -e lint          # code style
 Build the charm in this git repository using:
 
 ```shell
-tox run -e build-dev
+charmcraftcache pack
```

Comment:
**thought**: hum, I know that I already approved PRs with this specific change, but now that I slept on it, we do not introduce `charmcraftcache` at all in `CONTRIBUTING.md` or `README.md`. It is also unlikely that a new user would know about it from the Juju docs since it is still quite internal to our team.
Ultimately, this is but a minor grip, so I will not request changes from PRs with this docs change.

Reply:
I commented the same in https://github.com/canonical/pgbouncer-operator/pull/370#discussion_r1776869643

Only one repo has up-2-date preparation section at the moment :-(